### ISSUE TITLE
Skip all tags, when `skip_tags = ""`

### ISSUE THREAD

**Issue by hituzi-no-sippo** [orhun/git-cliff#63]
I'm sorry if the behavior this is a specification.

**Describe the bug**

Skip all tags, when `skip_tags = ""`.
Is this the expected behavior?

test script

**To Reproduce**

I've created a test script.

<details><summary>test script</summary>

```bash
#!/bin/bash
set -e

rm -rf /tmp/cliff/skip_tags_test
mkdir -p /tmp/cliff/skip_tags_test
cd /tmp/cliff/skip_tags_test

git init

git commit --allow-empty -m "chore: v0.0.1"
git tag v0.0.1

git commit --allow-empty -m "feat: v0.1.0-beta.1"
git tag v0.1.0-beta.1

git commit --allow-empty -m "feat: v0.1.0"
git tag v0.1.0

git cliff --init

echo '====================================='
echo 'skip_tags = "v0.1.0-beta.1" (Default)'
echo '====================================='
git cliff

sed -i 's/\(skip_tags = \)"v0.1.0-beta.1"/\1""/' cliff.toml
echo '====================================='
echo 'skip_tags = ""'
echo '====================================='
git cliff

sed -i 's/skip_tags.*//' cliff.toml
echo '====================================='
echo 'skip_tags is unset.'
echo '====================================='
git cliff
```

</details>

<details><summary>output</summary>

```
Initialized empty Git repository in /tmp/cliff/skip_tags_test/.git/
[main (root-commit) c57e04c] chore: v0.0.1
[main fe30e5e] feat: v0.1.0-beta.1
[main 9492956] feat: v0.1.0
 INFO  git_cliff > Saving the configuration file to "cliff.toml"
=====================================
skip_tags = "v0.1.0-beta.1" (Default)
=====================================
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2022-02-19

### Features

- V0.1.0

## [0.0.1] - 2022-02-19

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
=====================================
skip_tags = ""
=====================================
# Changelog

All notable changes to this project will be documented in this file.

<!-- generated by git-cliff -->
=====================================
skip_tags is unset.
=====================================
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2022-02-19

### Features

- V0.1.0

## [0.1.0-beta.1] - 2022-02-19

### Features

- V0.1.0-beta.1

## [0.0.1] - 2022-02-19

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
```

</details>

`skip_tags = ""`

```
# Changelog

All notable changes to this project will be documented in this file.

<!-- generated by git-cliff -->
```

**Expected behavior**

I expect don't skip all tags, as when `skip_tags` is unset.

`skip_tags` is unset.

```
# Changelog

All notable changes to this project will be documented in this file.

## [0.1.0] - 2022-02-19

### Features

- V0.1.0

## [0.1.0-beta.1] - 2022-02-19

### Features

- V0.1.0-beta.1

## [0.0.1] - 2022-02-19

### Miscellaneous Tasks

- V0.0.1

<!-- generated by git-cliff -->
```

**System (please complete the following information):**

- OS Ubuntu 20.04 LTS (WSL2)
- Project Version: [761669eb575cd3195554e2a5c680c62bcee4f99b](https://github.com/orhun/git-cliff/commit/761669eb575cd3195554e2a5c680c62bcee4f99b)

**Additional context**

Code that may be relevant.

https://github.com/orhun/git-cliff/blob/761669eb575cd3195554e2a5c680c62bcee4f99b/git-cliff/src/lib.rs#L124-L130

I have changed to the following code.

```rust
let skip = skip_regex.map(|r| r.is_match(name)).unwrap_or_default();
println!("{}", skip);
```

`println!("{}", skip);` output `true` always, when `skip_tags = ""`.

Thank you.

**Comment by orhun** [comment]
Hey, good catch! It should be fixed in ff1d981fd9a5dba26422f56582e06b3b463eb8a3 🐻

**Comment by hituzi-no-sippo** [comment]
Good job 👍

I've checked, it worked as expected.
Thank you 😄
